src/Workspaces/MSBuildTest/WorkspaceTestBase.cs

```diff
@@ -107,6 +107,10 @@ protected static FileSet GetSimpleCSharpSolutionFiles()
         protected static FileSet GetSimpleVisualBasicSolutionFiles()
         {
             return new FileSet(
+                (@"NuGet.Config", Resources.NuGet_Config),
```

Comment:
Th intent of these files is to make sure the projects are isolated from the directory they are deployed to. Given that seems like we also need an `.editorconfig` file that sets root to true so editor config discovery doesn't bite us.

Reply:
Agreed. I believe at one point we actually deployed a file during one of our powershell scripts, but I'm unable to find that now. I'm also not sure why each test has to specify these manually, since any test that _doesn't_ include them is automatically busted.